freshmaker/handlers/koji/rebuild_images_on_rpm_advisory_change.py

```diff
@@ -319,6 +320,9 @@ def _record_batches(self, batches, event, builds=None):
                     for content_set in image["content_sets"]:
                         if content_set in image["compose_sources"]:
                             continue
+                        # Exclude hidden content set.
+                        if re.search(".+(-hidden-).+", content_set):
```

Comment:
```suggestion
                        if content_set.endswith("-hidden-rpms"):
```

But this should probably go into Freshmaker configuration in our internal repo as a list of regexps, because it is an internal thing.

Reply:
I am not sure, do we have an external freshmaker service to deploy?

Reply:
No, but any internal and potentially sensitive data should be kept in our internal git repos.

Reply:
Ok. updated